src/lib/error-handler.js

```diff
@@ -0,0 +1,28 @@
+function errorMessage(msg, info) {
+  if (info) {
+    return `${msg} : additional info ${info}`
+  }
+
+  return msg;
+}
+
+function setVueErrorHandler(vue) {
+  vue.config.errorHandler = errorHandler
+}
+
+function errorHandler(err, _vm, info) {
+  if ((typeof err === 'object') && err.message) {
+    if (info) {
+      err.message = errorMessage(err.message, info);
+    }
+
+    throw err;
+  }
+
+  throw new Error(errorMessage(err, info));
+}
+
+export {
```

Comment:
Can you export the functions rather than an object that contains the functions

Reply:
👍